app/models/person.rb

```diff
@@ -15,4 +15,12 @@ def name
   def full_name
     "#{title} #{name}"
   end
+
+  def to_string
+    "#{title} #{first_name} #{last_name}"
+  end
```

Comment:
What is the difference between to_string and full_name in this case?

Reply:
I don't see a difference in value. But they are used in two different contexts. to_string is used to show the item in a searching result list. The other one is for the name only. It can be possible that the "search string" changes with the time (for example adding a chair to the string).

Reply:
You could then replace the content of to_string with a call to full_name, although it's probably also fine if it stays like that